erehwon/erehwon/settings/__init__.py

```diff
@@ -144,6 +144,8 @@
     os.path.join(BASE_DIR, "../static"),
 ]
 
+# AUTH_USER_MODEL = 'profiles.ErehwonUser'
```

Comment:
This PR is now only for django-registration, not custom_user implementation so ErehwonUser is not yet created.